### PR TITLE
Moving IE11 detection to a utils file

### PR DESCRIPTION
**Pull request by damianpm**
**What this PR does / why we need it**:

Extract IE 11 detection method to a utils file

**Which issue(s) this PR fixes** 

https://issues.jboss.org/browse/THREESCALE-2992

### CODE REVIEW

File: spec/javascripts/utilities/ie11Utils.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { isBrowserIE11 } from 'utilities/ie11Utils'
+
+describe('isBrowserIE11', () => {
+  it('should return false if browser is not IE11', () => {
```
Comment (josemigallas) [inline spec/javascripts/utilities/ie11Utils.spec.jsx]: I think the correct statement would be _it should return false if the user agent is other than Trident/7.0_.

Comment (damianpm) [reply spec/javascripts/utilities/ie11Utils.spec.jsx]: Updated!

File: app/javascript/src/utilities/ie11Utils.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const isBrowserIE11 = () => !!window.navigator.userAgent.match(/Trident\/7\./)
```
Comment (didierofrivia) [inline app/javascript/src/utilities/ie11Utils.js]: use DI here for easier and isolated testing, like:

```
 const isBrowserIE11 = (win = window) => !!win.navigator.userAgent.match(/Trident\/7\./)
```

then in the test you could inject the value you want.

Comment (damianpm) [reply app/javascript/src/utilities/ie11Utils.js]: Good idea, it's updated @didierofrivia !

**Pull request merged**